src/pwa/stores/base.js

```diff
@@ -1,10 +1,31 @@
 import { types, getParent, getEnv } from 'mobx-state-tree';
 
-export default types.model('Organization').actions(self => ({
-  addProcessors() {
-    const { h2r } = getParent(self, 1).theme;
-    const { processors } = getEnv(self).organization;
-
-    processors.forEach(proc => h2r.addProcessor(proc, 'low'));
-  },
-}));
+export default types.model('Organization').actions(self => {
+  const root = getParent(self, 1);
```

Comment:
Any advantage of using `root` as a scoped variable instead of a view, like we used to do?:
```js
.views(self => ({
  get root() { return getParent(self, 1); },
})
```
You can't mock it anymore.

Reply:
Yes, you are right. I'll change this.